ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1167,20 +1229,24 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
               </div>
             </OverlayPanel>
             <FlexRow>
-              <Dropdown style={{width: '14rem'}}
+              {this.state.fetchBillingAccountLoading ? <SpinnerOverlay overrideStylesOverlay={styles.spinner}/> :
```

Comment:
Thanks for adding the gif. I think the spinner is a bit missplaced - it is rendering off to the side currently. I would suggest instead that you render the spinner over the entire billing section, i.e. everything under the `WorkspaceEditSection` starting at L1223